act/qc/clean.py

```diff
@@ -10,10 +19,19 @@ def __init__(self, xarray_obj):
         self._obj = xarray_obj
 
     @property
-    def matched_qc_variables(self):
+    def matched_qc_variables(self, check_arm_syntax=True):
         """
         Find variables that are QC variables and return list of names.
 
+        Parameters
+        ----------
+        check_arm_syntax : boolean
+            ARM ueses a standard of starting all quality control variables
+            with "qc_". This is a more robust method of getting the quality
```

Comment:
The underscore is your issue here. In RST, an underscore is a hyperlink. This needs to be "qc\_".